vnext/Microsoft.ReactNative/QuirkSettings.idl

```diff
@@ -59,5 +59,11 @@ namespace Microsoft.ReactNative
         "window of that view to get focus as well. Set this setting to true to prevent focus of a blurred "
         "window when a view in that window is programmatically focused.")
     static void SetSuppressWindowFocusOnViewFocus(ReactInstanceSettings settings, Boolean value);
+
+    DOC_STRING(
+        "By default `react-native-windows` will use the new RuntimeScheduler."
```

Comment:
Is this new RuntimeScheduler coming from RN? Can that be expressed in the comment?
```suggestion
        "By default `react-native-windows` will use the new RuntimeScheduler from `react-native`."
```